mesa/model.py

```diff
@@ -57,6 +57,9 @@ def __new__(cls, *args: Any, **kwargs: Any) -> Any:
             # advance.
             obj._seed = random.random()  # noqa: S311
         obj.random = random.Random(obj._seed)
+        # TODO: Remove these 2 lines just before Mesa 3.0
+        obj._steps = 0
+        obj._time = 0
```

Comment:
Could you remind me why we need to remove these lines? Seems like the time.

Reply:
It's because they are already specified in `__init__`. This was a hack to people who forget to `super().__init__` their model.